algorithms/CSharp/src/Search/minima-maxima.cs

```diff
@@ -0,0 +1,100 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Algorithms.Search
+{
+    public class MinimaMaxima
+    {
+        public static void Main()
+        {
+            var numbers = new List<int>
+            {
+                3,1,2,5,6,7,4,6,9,10
+            };
+
+            var lmins = GetLocalMinima(numbers);
+            var lmaxs = GetLocalMaxima(numbers);
+
+            foreach (var n in lmins)
+            {
+                Console.WriteLine($"Local Minima: {n}");
+            }
+
+            foreach (var n in lmaxs)
+            {
+                Console.WriteLine($"Local Maxima: {n}");
+            }
```

Comment:
Please call the method with a new name
```suggestion
            var minimas= FindMinimas(numbers);
            var maximas= FindMaximas(numbers);

            foreach (var minima in minimas)
            {
                Console.WriteLine($"Local Minima: {minima}");
            }

            foreach (var maxima in maximas)
            {
                Console.WriteLine($"Local Maxima: {maxima}");
            }
```